lib/apiScenario/gen/staticTestScenarioGenerator.ts

```diff
@@ -68,6 +68,10 @@ export class StaticApiScenarioGenerator {
     const paths = await this.swaggerAnalyzer.getOperationListPath();
     for (const it of paths) {
       const example = Object.values(it.get?.["x-ms-examples"] || {});
+      if (example[0] === undefined) {
+        console.log("can not found example for operations-list");
```

Comment:
Can we also output the api path info?

Reply:
api path is operations  list.

Reply:
No, I mean the swagger info. Otherwise, we don't know which one miss the operation_list.